setup.cfg

```diff
@@ -48,16 +48,16 @@ dev =
     myst-parser
     numpydoc
     pre-commit
+    pydata-sphinx-theme
```

Comment:
@mrakitin here's the theme dependency, it moved from a `requirements.txt` file into a `[dev]` extra in a previous PR